### DISCUSSION TITLE
AxMath for Windows Stoppped Working today

### DISCUSSION THREAD

**Discussion by tommck-889161** [loongmxbt/AxMath#23] [Q&A]
Been using AxMath for a couple of years. Today (11.25.24) it stopped working. AxMath opens but wont start completely. Just get the spinning windows busy circle. Any help would be appreciated.

**Comment by tommck-889161** [comment]
I re-installed new version and that fixed AxMath in Word.

**Comment by stoneyakoya** [comment]
コード debt の定量調査を追加しました → [#1417 コード debt・汚さの規模感](https://github.com/niigata-bioinfo-st/monju-medical/discussions/1417)
